src/adagrams.js

```diff
@@ -1,15 +1,125 @@
 export const drawLetters = () => {
-  // Implement this method for wave 1
+  const letterFreq = {
+    A: 9,
+    B: 2,
+    C: 2,
+    D: 4,
+    E: 12,
+    F: 2,
+    G: 3,
+    H: 2,
+    I: 9,
+    J: 1,
+    K: 1,
+    L: 4,
+    M: 2,
+    N: 6,
+    O: 8,
+    P: 2,
+    Q: 1,
+    R: 6,
+    S: 4,
+    T: 6,
+    U: 4,
+    V: 2,
+    W: 2,
+    X: 1,
+    Y: 2,
+    Z: 1,
+  };
+  const drawn = [];
+  while (drawn.length < 10) {
+    let letters = Object.keys(letterFreq);
+    let randIndex = Math.floor(Math.random() * letters.length);
+    let randLetter = letters[randIndex];
+    let remainingLetterValue = letterFreq[randLetter];
+    if (remainingLetterValue > 0) {
+      drawn.push(randLetter);
+      letterFreq[randLetter] -= 1;
+    }
+  }
```

Comment:
This has a very subtle bug.  

Each letter drawn has a 1/26th chance of being drawn each time.  So the odds of you drawing a Z on your first draw is the same as an E.